### DISCUSSION TITLE
Need the realm 0.87.5 jar or aar file

### DISCUSSION THREAD

**Discussion by sudarshanharbinger** [realm/realm-java#7902] [General]
Hi, can you please provide me realm 0.87.5 version aar file or jar file

**Comment by reregaga** [comment]
@sudarshanharbinger You can build it yourself
Sources:
https://github.com/realm/realm-java/archive/refs/tags/v0.87.5.zip
https://github.com/realm/realm-java/archive/refs/tags/v0.87.5.tar.gz
Guide:
https://github.com/realm/realm-java/blob/v0.87.5/README.md#building-realm
